src/ts/sign.ts

```diff
@@ -165,15 +165,23 @@ export async function signOrder(
   owner: Signer,
   scheme: EcdsaSigningScheme,
 ): Promise<EcdsaSignature> {
+  const signature = await ecdsaSignTypedData(
+    scheme,
+    owner,
+    domain,
+    { Order: ORDER_TYPE_FIELDS },
+    normalizeOrder(order),
+  );
+  // Passing the signature over split/join to normalize the `v` byte.
+  // Some wallets do not pad it with `27`, which causes a signature failure
+  // `splitSignature` pads it if needed, and `joinSignature` simply puts it back together
+  const data = ethers.utils.joinSignature(
+    ethers.utils.splitSignature(signature),
+  );
```

Comment:
can this be added to `ecdsaSignSignature` instead so that both `signOrder` and `signOrderCancellation` get the fix. Otherwise we would have to duplicate this for order cancellation signing.

Reply:
Done